packages/atomic/src/components/atomic-no-results-v1/atomic-no-results.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  buildHistoryManager,
+  HistoryManager,
+  HistoryManagerState,
+  buildSearchStatus,
+  SearchStatus,
+  SearchStatusState,
+  QuerySummary,
+  QuerySummaryState,
+  buildQuerySummary,
+} from '@coveo/headless';
+import {Component, h, Prop, State} from '@stencil/core';
+import {
+  Bindings,
+  BindStateToController,
+  InitializeBindings,
+} from '../../utils/initialization-utils';
+import Lupa from '../../images/lupa-frame.svg';
+
+/**
+ * The `atomic-no-results` component displays search tips and a "Cancel last action" button when there are no results. Any additional content slotted inside of its element will be displayed as well.
+ *
+ * @part cancel-button - The "Cancel last action" button.
+ * @part no-results - The text indicating that no results were found for the search.
+ * @part search-tips - The search tips to help the user correct the query.
+ * @part highlight - The highlighted query.
+ */
+@Component({
+  tag: 'atomic-no-results-v1', //TODO remove v1
+  styleUrl: 'atomic-no-results.pcss',
+  shadow: true,
+})
+export class AtomicNoResults {
+  @InitializeBindings() public bindings!: Bindings;
+  public searchStatus!: SearchStatus;
+  public history!: HistoryManager;
+  public querySummary!: QuerySummary;
+
+  @BindStateToController('searchStatus')
+  @State()
+  private searchStatusState!: SearchStatusState;
+  @BindStateToController('history')
+  @State()
+  private historyState!: HistoryManagerState;
+  @BindStateToController('querySummary')
+  @State()
+  private querySummaryState!: QuerySummaryState;
+  private strings = {
```

Comment:
this is a relic from the past, not necessary, you can use `this.bindings.i18n.t('search-tips-v1')` directly